.scripts/common.ts

```diff
@@ -112,14 +112,31 @@ function isPackageFolderPath(folderPath: string, packagesToIgnore: string[]): bo
   return result;
 }
 
+function shouldProcessFolderPath(folderPath: string, folderNamesToIgnore: string[]): boolean {
+  if (!contains(folderNamesToIgnore, getName(folderPath))) {
+    // If current path is not in folder to ignore list then process it.
+    return true;
```

Comment:
I think this returns true for the track 1 perf test packages since it isn't a rush package.
Track 1 perf test packages should also be ignored.
Logged a new issue to handle this case #14304.

Reply:
@HarshaNalluru : Do we need any more change in this PR or is it good to merge?

Reply:
This looks good. (perf track 1 can be handled in a separate PR.)

But I think if you can elaborate in comments with examples on what gets filtered at each step, it would help a lot.